popup.css

```diff
@@ -1,112 +1,98 @@
-@font-face {
-  font-family: "Roboto";
-  src: url("Roboto/Roboto-Bold.ttf") format("truetype");
-}
-
-body {
-  background: #242a36;
-  width: 340px;
-  padding: 1em;
-}
-
-.btn {
-  width: 100% !important;
-  padding: 5px;
-  height: 41px;
-  border-radius: 6px;
-  font-size: 20px;
-  overflow: hidden;
-  font-family: "Roboto", sans-serif;
-  border-width: 0;
-  outline: none;
-  margin-bottom: 0.4em;
-  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.6);
-  background-color: #2ecc71;
-  color: #ecf0f1;
-  cursor: pointer;
-
-  transition: background-color 0.3s;
-}
-
-.btn:hover,
-.btn:focus {
-  background-color: #27ae60;
-}
-
-.btn > * {
-  position: relative;
-}
-
-.btn span {
-  display: block;
-  padding: 12px 24px;
-}
-
-.btn:active:before {
-  width: 120%;
-  padding-top: 120%;
-
-  transition: width 0.2s ease-out, padding-top 0.2s ease-out;
-}
-
-#message {
-  margin: 3.2em 0em 1em 0.5em;
-  font-family: "Roboto", sans-serif;
-  font-style: normal;
-  font-weight: bold;
-  padding-left: 0.3em;
-  padding-right: 0.3em;
-  font-size: 16px;
-  line-height: 21px;
-  align-items: center;
-  color: white;
-}
-
-.options {
-  float: right;
-  margin-right: 0.1em;
-}
-
-#activeContol {
-  float: left;
-}
-.main {
-  clear: both;
-}
-.content {
-  font-style: normal;
-  font-weight: bold;
-  font-size: 18px;
-  line-height: 21px;
-  display: flex;
-  align-items: center;
-  color: #ffffff;
-}
-/* Styles, not important */
 *,
-*:before,
-*:after {
-  box-sizing: border-box;
-}
-
-h2 {
-  font-weight: normal;
-}
-
-.btn.orange {
-  background-color: #e67e22;
-}
-
-.btn.orange:hover,
-.btn.orange:focus {
-  background-color: #d35400;
-}
-
-.btn.red {
-  background-color: #e74c3c;
-}
-
-.btn.red:hover,
-.btn.red:focus {
-  background-color: #c0392b;
-}
```

Comment:
@fabcodingzest Your PR has too many changes in the files irrelevant to the issue, like why did you change all these styles from popup.css.

Reply:
Oh I changed the popup design as you asked in issue #35 so I made my other classes according to how I changed the markup for popup a bit to use flexbox. I pretty much added 6 files I guess 3 for options nd 3 for popup - it runs fine thou as I tried it locally from my branch .

Tell me if u need any changes i would make them I hope.

Reply:
Hey @prateek3255 I am waiting for your feedback ! 😅